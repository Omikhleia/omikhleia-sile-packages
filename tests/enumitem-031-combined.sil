\begin[papersize=a6, class=book]{document}
\script[src=packages/enumitem]
\font[family=Libertinus Serif]% Default Gentium lacks the white circle.
\nofolios
\neverindent
% Test case 031: nested enumerate and itemize levels, all folding back

\begin{enumerate}
    \item{Level 1 Enum}
    \begin{itemize}
        \item{Level 2 Item}
        \begin{enumerate}
            \item{Level 3 Enum}
            \begin{itemize}
                \item{Level 4 Item}
                \begin{enumerate}
                    \item{Level 5 Enum}
                    \begin{itemize}
                        \item{Level 6 First Item}
                    \end{itemize}
                    \item{Level 5 Enum}
                \end{enumerate}
                \item{Level 4 Item}
            \end{itemize}
            \item{Level 3 Enum}
        \end{enumerate}
        \item{Level 2 Item}
    \end{itemize}
    \item{Level 1 Enum}
\end{enumerate}

\end{document}
