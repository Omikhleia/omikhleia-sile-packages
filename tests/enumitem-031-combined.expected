Set paper size 	297.6377985	419.5275636
Begin page
Mx 	29.7039
My 	55.6452
Set font 	Libertinus Serif;10;400;;normal;;LTR
T	18 w=4.6500	(1)
Mx 	34.3539
T	15 w=2.2000	(.)
Mx 	44.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	69.4033
T	18 w=4.6500	(1)
Mx 	77.0726
T	38 79 86 78 w=24.2000	(Enum)
Mx 	50.4489
My 	67.6452
T	1768 w=3.5100	(•)
Mx 	59.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	84.4021
T	19 w=4.6500	(2)
Mx 	92.0703
T	42 85 70 78 w=18.5000	(Item)
Mx 	64.7039
My 	79.6452
T	74 w=2.7100	(i)
Mx 	67.4139
T	15 w=2.2000	(.)
Mx 	79.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	104.3989
T	20 w=4.6500	(3)
Mx 	112.0639
T	38 79 86 78 w=24.2000	(Enum)
Mx 	85.4489
My 	91.6452
T	2222 w=3.5100	(◦)
Mx 	94.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	119.3977
T	21 w=4.6500	(4)
Mx 	127.0615
T	42 85 70 78 w=18.5000	(Item)
Mx 	99.7039
My 	103.6452
T	66 w=4.5700	(a)
Mx 	104.2739
T	10 w=2.9800	())
Mx 	114.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	139.3945
T	22 w=4.6500	(5)
Mx 	147.0551
T	38 79 86 78 w=24.2000	(Enum)
Mx 	119.4639
My 	115.6452
T	1753 w=5.4800	(–)
Mx 	129.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	154.3907
T	23 w=4.6500	(6)
Mx 	162.0475
T	39 74 83 84 85 w=18.3400	(First)
Mx 	183.3942
T	42 85 70 78 w=18.5000	(Item)
Mx 	99.7039
My 	127.6452
T	67 w=4.9300	(b)
Mx 	104.6339
T	10 w=2.9800	())
Mx 	114.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	139.3945
T	22 w=4.6500	(5)
Mx 	147.0551
T	38 79 86 78 w=24.2000	(Enum)
Mx 	85.4489
My 	139.6452
T	2222 w=3.5100	(◦)
Mx 	94.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	119.3977
T	21 w=4.6500	(4)
Mx 	127.0615
T	42 85 70 78 w=18.5000	(Item)
Mx 	64.7039
My 	151.6452
T	74 74 w=5.4200	(ii)
Mx 	70.1239
T	15 w=2.2000	(.)
Mx 	79.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	104.3989
T	20 w=4.6500	(3)
Mx 	112.0639
T	38 79 86 78 w=24.2000	(Enum)
Mx 	50.4489
My 	163.6452
T	1768 w=3.5100	(•)
Mx 	59.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	84.4021
T	19 w=4.6500	(2)
Mx 	92.0703
T	42 85 70 78 w=18.5000	(Item)
Mx 	29.7039
My 	175.6452
T	19 w=4.6500	(2)
Mx 	34.3539
T	15 w=2.2000	(.)
Mx 	44.7039
Mx 	5.2800
Mx 	4.4000
Mx 	4.8900
Mx 	4.4700
Mx 	2.6400
T	45 a=5.2800 70 a=4.4700 87 a=4.9700 70 a=4.4700 77 a=2.6400	(Level)
Mx 	69.4033
T	18 w=4.6500	(1)
Mx 	77.0726
T	38 79 86 78 w=24.2000	(Enum)
End page
Finish
